### PR TITLE
Pagination events

### PR DESCRIPTION
**Pull request by idanpeduiem**

### CODE REVIEW

File: client/src/components/Events.tsx [hunk 1]
```diff
@@ -1,32 +1,88 @@
...
+        onScroll={onScroll}
+        sx={{ height: "inherit", overflowY: "auto" }}
+      >
+        {events.map((event) => (
+          <Grid key={event.id} item sm={4} md={3}>
+            <TicketCard />
+          </Grid>
+        ))}
+      </GridHiddenScroll>
+      {/*{fetching && (*/}
```
Comment (thcr1) [inline client/src/components/Events.tsx]: remove commented code

File: server/src/server.ts [hunk 2]
```diff
@@ -16,22 +16,6 @@ import setUpMongo from "../mongo/mongoDbManager";
 
   setUpMongo();
 
-  app.use(cookieParser());
-  
-  app.use(cors({
```
Comment (thcr1) [inline server/src/server.ts:21]: you are right that it shouldn't be like that, but we still need cors

**Review by avivtsadik — APPROVED** [review]

File: client/src/components/Events.tsx (hunk 1 above)
Comment (idanpeduiem) [reply client/src/components/Events.tsx]: removed

File: server/src/server.ts (hunk 2 above)
Comment (idanpeduiem) [reply server/src/server.ts:21]: yoga add these cors by default, i dont think we need more cors then that but maybe i miss something so let me know if more cors needed
image

docs:
https://the-guild.dev/graphql/yoga-server/docs/features/cors

**Pull request merged**